packages/core/src/__tests__/ceramic-pinning.test.ts

```diff
@@ -1,8 +1,8 @@
-import { jest } from '@jest/globals'
+import { jest, describe, expect, beforeEach, afterEach } from '@jest/globals'
```

Comment:
I noticed that these imports were being used, even though they weren't explicitly imported. Is there a particular reason why are they imported explicitly now?

Reply:
For us absolute most of the times it does not matter.

We had some troubles in the past when running the test files with implicit `describe`, `expect`, etc. If I remember correctly, it was due to the fact TS compiler (in some non-trivial mode) could not actually figure out the types. If we are to move away from babel-jest, the issue will reappear.

And, you could experience an inconvenience when an editor decides it is mocha or something similar.